### PR TITLE
docs(id): improve i18n/id-ID.js

### PR DESCRIPTION
**Pull request by arsandev**

### CODE REVIEW

File: i18n/id-ID.js [hunk 1]
```diff
@@ -16,13 +16,13 @@ module.exports = {
   header: {
     links: [
       {
-        name: 'Guides',
+        name: 'Panduan',
```
Comment (febrihidayan) [inline i18n/id-ID.js:19]: Halo mas @arsandev, bagaimana kalau `Panduan` menjadi `Panduan's` seperti itu mas. Soalnya nanti setelah ini di commit akan ada menu Panduan dan Panduan, padahal slug berbeda

Comment (arsandev) [reply i18n/id-ID.js]: i see. oke, saya commit lagi

Comment (febrihidayan) [reply i18n/id-ID.js]: Baik mas

**Review by febrihidayan — APPROVED** [review]

**Comment by debs-obrien** [comment]
Hi @febrihidayan  I am making you an official reviewer of the indonesian . You will receive an invite to have access to the repo and then you should be able to merge the indonesian files when you think they are ready for merging. let me know how it goes. thanks for your great work and contributions 😊

**Comment by febrihidayan** [comment]
Thanks @debs-obrien, I will try to do better 😀

**Review by debs-obrien — APPROVED** [review]

**Comment by debs-obrien** [comment]
@febrihidayan let me know if you have permissions to merge this

**Comment by febrihidayan** [comment]
@debs-obrien It's ready to be updated

**Comment by arsandev** [comment]
thanks for review and merging, and good luck @febrihidayan !

**Comment by febrihidayan** [comment]
@arsandev belum di tambahkan ke master, mas. Seharusnya tidak menutup tarik permintaan ini

**Comment by debs-obrien** [comment]
@febrihidayan you should now see a merge button that you can click to actually merge the pr. not just merge branch to master but actually merge. if you can't see that button let me know so I can fix it. one all checks are passed you should be able to see a merge button

**Comment by febrihidayan** [comment]
@debs-obrien I have clicked on the update branch button. But, the merge button doesn't work. Should I click the update branch button again?

**Comment by debs-obrien** [comment]
try it again as soon as those checks have finished. it seems something else got merged while you were updating the branch

**Comment by febrihidayan** [comment]
@arsandev Terimakasih 😀
@debs-obrien Thanks 😀, This is very nice

**Comment by debs-obrien** [comment]
you have power now @febrihidayan 💪 and I trust you will do a great job

**Pull request merged**